packages/web/src/components/search/CategorySearch.js

```diff
@@ -825,13 +824,14 @@ class CategorySearch extends Component {
 	};
 
 	getComponent = (downshiftProps = {}) => {
-		const { error, isLoading } = this.props;
+		const { error, isLoading, aggregationData } = this.props;
 		const { currentValue } = this.state;
 		const data = {
 			error,
 			loading: isLoading,
 			downshiftProps,
 			data: this.parsedSuggestions,
+			aggregationData,
```

Comment:
@ShahAnuj2610 Do we return `aggregationData` in `onData` method too?

Reply:
yes, but `onData` is only for `ReactiveComponent` and `ReactiveList`

Reply:
I mean `onSuggestions` in that case.

Reply:
@ShahAnuj2610 I checked it and found that we don't have `onData` in search components, so you can leave it for now.